akka-actor/src/main/mima-filters/2.6.5.backwards.excludes/ActorRefOps.excludes

```diff
@@ -0,0 +1,2 @@
+# Replace actorRef2Scala with value class.
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.package.actorRef2Scala")
```

Comment:
I pretty sure that's not safe to filter, any code using `!` compiled against 2.6.4- would stop working.

Reply:
Yes... Missed that.